src/app/InteractionModelEngine.h

```diff
@@ -199,10 +197,8 @@ class InteractionModelEngine : public Messaging::ExchangeDelegate, public Comman
 
     void OnDone(CommandHandler & apCommandObj) override;
 
-    CHIP_ERROR OnUnknownMsgType(Messaging::ExchangeContext * apExchangeContext, const PayloadHeader & aPayloadHeader,
-                                System::PacketBufferHandle && aPayload);
     CHIP_ERROR OnInvokeCommandRequest(Messaging::ExchangeContext * apExchangeContext, const PayloadHeader & aPayloadHeader,
-                                      System::PacketBufferHandle && aPayload);
+                                      System::PacketBufferHandle && aPayload, Protocols::InteractionModel::Status & aStatus);
```

Comment:
The semantics around this aStatus value need to be documented.  It's entirely non-obvious from this (undocumented API) that the caller pre-sets it to failure and the callee is expected to set it to success if it does not want an automatic status response message to be sent.

Same for the other APIs here.

Reply:
updated, thanks